src/Credits.py

```diff
@@ -0,0 +1,70 @@
+import curses
+
+# Interface Class for Credits
+class Credits:
+    def __init__(self, screen):
+        self.screen = screen
+        self.top = "--- CREDITS ---"
+        self.header = ["Projektmanager", "Vertriebsmanager", "Technologiemanager", "Qualitaetsmanager"]
+        self.projektmanagement = ["Knut Z.", "Max C."]
+        self.vertriebsmanagement = ["Lukas S.", "Pascal T.", "Fynn B."]
+        self.technologiemanagement = ["Dennis K.", "Mostafa K.", "Tim G."]
+        self.qualitaetsmanagement = ["Tobias A.", "Michael L.", "Oliver O."]
+
+    def print_list(self, win, row, col, list):
+        for x in list:
+            win.addstr(row, col, x)
+            for i in range(0, len(list)):
+                row += 1
```

Comment:
Die ganze Methode `print_list` sollte nicht auf einer Klasse definiert sein, sondern gelöst davon, da du die Klasse selbst in der Methode nicht verwendest.